dotnet/src/Extensions/Planning.SequentialPlanner/SequentialPlanParser.cs

```diff
@@ -40,16 +40,29 @@ internal static class SequentialPlanParser
     /// </summary>
     internal const string AppendToResultTag = "appendToResult";
 
+    internal static Func<string, string, ISKFunction?> GetFunction(SKContext context)
+    {
+        return (skillName, functionName) =>
+        {
+            if (context.Skills!.TryGetFunction(skillName, functionName, out var skillFunction))
+            {
+                return skillFunction;
+            }
+
+            return null;
+        };
+    }
+
     /// <summary>
     /// Convert a plan xml string to a plan.
     /// </summary>
     /// <param name="xmlString">The plan xml string.</param>
     /// <param name="goal">The goal for the plan.</param>
-    /// <param name="context">The semantic kernel context.</param>
+    /// <param name="GetFunction">The function to get a skill function.</param>
     /// <param name="allowMissingFunctions">Whether to allow missing functions in the plan on creation.</param>
     /// <returns>The plan.</returns>
     /// <exception cref="PlanningException">Thrown when the plan xml is invalid.</exception>
-    internal static Plan ToPlanFromXml(this string xmlString, string goal, SKContext context, bool allowMissingFunctions = false)
+    internal static Plan ToPlanFromXml(this string xmlString, string goal, Func<string, string, ISKFunction?> GetFunction, bool allowMissingFunctions = false)
```

Comment:
Even if it's delegate, I would still use camelCase for function parameter name - `getFunction`.

Reply:
I'm surprised our formatting rules didn't flag this.